### ISSUE TITLE
Indicators for A1: (meta)data are retrievable by their identifier using a standardised communication protocol

### ISSUE THREAD

**Issue by makxdekkers** [RDA-FAIR/FAIR-data-maturity-model-WG#19]
![image](https://user-images.githubusercontent.com/7534601/60016913-d8967080-9687-11e9-9289-8c0dcd60345b.png)

**Comment by makxdekkers** [comment]
Points raised in online meeting 3 on 18 June 2019

- There is a slight difference between machine-readable and machine-understandable. The latter is needed for autonomous processing. That point could include a subset of indicators for actionability, readability, understandability, interpretability.
- What is an acceptable language for knowledge representation?
- A formal syntax (e.g. XML) should be one of the levels of the ‘format’ indicator.
- Community-standard format and proprietary format might overlap. If the indicator is maintained, there is still a difference for data and metadata. 
- It might be better to distinguish ‘syntax’ versus ‘semantic’.
- ‘Open’ should replace ‘Community-standard’ as opposed to ‘Proprietary’.
- In some disciplines, researchers only use proprietary formats. If the indicator remains, it means that some of the digital objects would never become FAIR. There may be a preference for open formats, but it is simply not always available.

**Comment by makxdekkers** [comment]
Points raised in online meeting 3 on 18 June 2019

- The protocol has to be transparent (i.e. people should be able to find out what it is).
- Manual access via the researcher needs to be clarified and broadened (e.g. through a data access committee, data stewards, etc.).
- ‘Standardised’ should be part of the indicators.

**Comment by RDA-FAIR** [comment]
![image](https://user-images.githubusercontent.com/47179213/60807972-7bbab000-a187-11e9-8c07-f50a997d9a35.png)

![image](https://user-images.githubusercontent.com/47179213/60807977-7d847380-a187-11e9-83b1-a1185ab4913b.png)


Here is a new version of the indicators!

**Comment by RDA-FAIR** [comment]
Please find the current version of the indicator(s) and their respective maturity levels for this FAIR principle. Indicators and maturity levels will be presented, as they stand, to the next working group meeting for approval.  In the meantime, any comments are still welcomed.

The editorial team will now concentrate on weighing and prioritizing these indicators. More information soon.

![image](https://user-images.githubusercontent.com/47179213/62354448-316dea00-b50d-11e9-91ad-37ef3bdbd1f0.png)

**Comment by kitchenprinzessin3880** [comment]
1. A1-01M - access conditions - is this refers to access status such as private, public, protected?

2. A1-02M - Identifier resolve --> resolve**s**

**Comment by kitchenprinzessin3880** [comment]
A1-02D - what does 'human interaction' refer to?

**Comment by makxdekkers** [comment]
Human interaction would be, for example, that the person who wants to access the data has to install some special software to access the data.

**Comment by kitchenprinzessin3880** [comment]
> Human interaction would be, for example, that the person who wants to access the data has to install some special software to access the data.

ok noted.

recently our repository (pangaea) involved in curating social science data; the workflow is abit different in the sense that users can apply an access to datasets using an online form available through the pangaea system. once the application is approved, they need to visit the data bank  (building) managed by other centers to get a copy of the data. so it this relates to A1-01D?

**Comment by rwwh** [comment]
Increasing FAIRness will demand more than just "not having to install special software". 

Every piece of software now can download data from the web, but if a piece of software needs to go through 3 levels of human-readable-only menus on a web site in order to find the link to the data, that is also a problem. Currently every repository software has its own search and retrieve API. Reducing those differences is also part of A1.

**Comment by rwwh** [comment]
@kitchenprinzessin3880 "access conditions" (I know the top level as "restricted" to specific use cases and users /"shared" within a project group / "public") become interesting when the data are restricted. For example, some hospital data may only have sufficient content for use in scientific studies on Lung Cancer. In the future, being able to encode these kind of conditions in a machine readable way will make the data more FAIR because a machine can decide autonomously whether it makes sense to request access.

**Comment by kitchenprinzessin3880** [comment]
> being able to encode these kind of conditions in a machine readable way will make the data more FAIR because a machine can decide autonomously whether it makes sense to request access.
+1 we need a simple list of access (public, private...)

**Comment by makxdekkers** [comment]
Peter Wittenburg at https://docs.google.com/spreadsheets/d/1mkjElFrTBPBH0QViODexNur0xNGhJqau0zkL4w8RRAw/edit?disco=AAAADadg7v8

On A1-03D:
We should not use the term "file" here, since this often is not the reality. Better speak about Digital Object which can be a file, data from a cloud container or extracted from a database.

**Comment by makxdekkers** [comment]
Peter, "_digital object_" instead of "_file_" may indeed be better.

**Comment by keithjeffery** [comment]
I would like to see more discusion on whether the *intention* of the FAIR principles was manual access to downloadable datasets

**Comment by makxdekkers** [comment]
Carole Goble wrote at https://docs.google.com/spreadsheets/d/1mkjElFrTBPBH0QViODexNur0xNGhJqau0zkL4w8RRAw/edit#gid=0

I agree - identifiers.org resolves to entries in a database that are accessible through an API or through a web site landing page. Please do not use "data file" as just meaning "data" - it has a very definite meaning

**Comment by makxdekkers** [comment]
@SusannaSansone wrote at https://docs.google.com/spreadsheets/d/1mkjElFrTBPBH0QViODexNur0xNGhJqau0zkL4w8RRAw/edit#gid=0

Agreed. Also for other DOs such as (community) standards, https://fairsharing.org DOIs resolve to a record describing that standards, where to access it, which repositories implement it etc.

**Comment by makxdekkers** [comment]
@keithjeffery 

> I would like to see more discusion on whether the _intention_ of the FAIR principles was manual access to downloadable datasets

Issue is raised for discussion at the online meeting on 12 September 2019, slide 26 in the presentation at https://www.rd-alliance.org/workshop-4.

**Comment by bahimc** [comment]
Dear contributors,
 
Below you can find the indicators and their maturity levels in their current state as a result of the above discussions and workshops.
 
![image](https://user-images.githubusercontent.com/47179213/66298945-dd0f2e00-e8f2-11e9-9a8b-4613a84e5a8b.png)

![image](https://user-images.githubusercontent.com/47179213/66299101-265f7d80-e8f3-11e9-9888-d3b3694d97bc.png)
 
**Please note that this thread is going to be closed**, within a short period of time.  The current state of the indicators, as of early October 2019, is now frozen, with the exception of the indicators for the principles that are concerned with ‘richness’ of metadata (F2 and R1). The current indicators will be used for the further steps of this WG, which are prioritisation and scoring.  Later on, they will be used in a testing phase where owners of evaluation approaches are going to be invited to compare their approaches (questionnaires, tools) against the indicators. The editorial team, in consultation with the Working Group, will define the best approach to test the indicators and evaluate their soundness. As such, the current set of indicators can be seen as an ‘alpha version’. In the first half of 2020, the indicators may be revised and improved, based on the results of the testing. **If you have any further comments, suggestions regarding that specific discussion, please share them with us. Besides, we invite you to have a look at the following two sets of issues.**
 
**Prioritisation**
 
[• Indicators prioritisation for Findability
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/30)[• Indicators prioritisation for Accessibility](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/31)
[• Indicators prioritisation for Interoperability
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/32)[• Indicators prioritisation for Reusability ](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/33)
 
**Scoring** 
 
[• Indicators for FAIRness | Scoring
](https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/issues/34) 
We thank you for your valuable input!
